rmgweb/main/templates/index.html

```diff
@@ -14,38 +14,57 @@
-Flux diagram for the pyrolysis of 1,3-hexadiene, an example model generated 
-with RMG, showing the net carbon flux at an instant near the end of the 
+Flux diagram for the pyrolysis of 1,3-hexadiene, an example model generated
+with RMG, showing the net carbon flux at an instant near the end of the
 simulation.
 </p>
 
-<p>You are now visiting the interactive website for RMG, featuring web tools to
+<p>
+You are now visiting the interactive website for RMG, featuring web tools to
 help you visualize RMG-generated models, as well as tools to view and search the RMG
 databases for thermodynamics, transport, and kinetics data.
+</p>
+
+<p>
+You can obtain a copy of the actual RMG software through the links below:
+</p>
 
-<p>You can obtain a copy of the actual RMG software through the links below:
-<P><strong>Latest Stable Version: {{version}} </strong>
- (<a href="http://reactionmechanismgenerator.github.io/RMG-Py/users/rmg/installation/index.html">Download and Install</a>)
 <p>
-<P><strong>Source Code</strong>:
- <a href="http://www.github.com/ReactionMechanismGenerator/RMG-Py">RMG-Py Github Repository</a>
-<P>
+<strong>Latest Stable Version: {{version}}</strong>
+(<a href="http://reactionmechanismgenerator.github.io/RMG-Py/users/rmg/installation/index.html">Download and Install</a>)
+</p>
 
-RMG is a free, open-source software package (distributed under the MIT/X11 license). 
-The software is developed in the <a href="http://cheme.scripts.mit.edu/green-group/">William 
+<p>
+<strong>Source Code</strong>:
+<a href="http://www.github.com/ReactionMechanismGenerator/RMG-Py">RMG-Py Github Repository</a>
+</p>
+
+<p>
+RMG is a free, open-source software package (distributed under the MIT/X11 license).
+The software is developed in the <a href="http://cheme.scripts.mit.edu/green-group/">William
 H. Green</a> research group in the <a href="http://web.mit.edu/cheme/">
-Department of Chemical Engineering</a> at <a href="http://web.mit.edu/">MIT</a> and in 
-the <a href="http://www.northeastern.edu/comocheng/">Richard H. West</a> research group 
+Department of Chemical Engineering</a> at <a href="http://web.mit.edu/">MIT</a> and in
+the <a href="http://www.northeastern.edu/comocheng/">Richard H. West</a> research group
 at the <a href="http://www.che.neu.edu">Department of Chemical engineering</a> at
- <a href="http://www.northeastern.edu">Northeastern University</a></p>
- <P>
-Please post comments or concerns about the website to 
+<a href="http://www.northeastern.edu">Northeastern University</a>
+</p>
+
+<p>
+Please post comments or concerns about the website to
 <a href="http://www.github.com/ReactionMechanismGenerator/RMG-website/issues">our issues page</a>
 or email the RMG Development Team at <a href="mailto:rmg_dev@mit.edu">rmg_dev@mit.edu</a>
+</p>
+
+<p>
+Please subscribe the <a href="http://mailman.mit.edu/mailman/listinfo/rmg-users">mailing list</a> to receive the latest updates
+of RMG software and workshop
```

Comment:
"subscribe to"

"RMG users mailing list"

"updates about the RMG software and other news."